### DISCUSSION TITLE
2023 WTF Academy国库盈余分配讨论

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#615] [General]
最近我在写年度总结，整理WTF Academy国库的时候发现截止2023年底，我们有约 65,000 u的盈余。其中大部分是$op（来自retropgf），虽然一直有花，但是随着牛市来了钱越花越多。

<img width="994" alt="截屏2024-01-03 下午5 09 18" src="https://github.com/AmazingAng/WTF-Solidity/assets/14728591/3f4baa12-74e5-486f-88a4-eec1349ac2f4">

2023年WTF Academy的目标是收支平衡，所以我们决定用50%的资金（30,000 u）来回溯性奖励WTF贡献者们。需要大家讨论如何分配。目前WTF贡献者中呼声比较高的方案：

**方案1.** 定制WTF周边（衣服/帽子/贴纸/水杯/电脑包/袜子/社区头像）发给贡献者，增加归属感。这个方案的难点就是定制需要设计资源还有和商家沟通成本。

**方案2.** 发钱，69%按贡献时间分，31%阳光普照平分给贡献者。这个方案的难点就是可靠的收集每个贡献者的贡献时间。

大家可以在这个帖子下给出你的建议，分配方案需要可量化，且兼顾实施效率。

另外，在新年开始，我还需要贡献者帮我两件事：

1. 如果2023年有社区任务奖励漏发，请及时联系我。因为会计由我一人兼任，很容易漏发，大家多多包含。
2. 如果你给WTF Academy进行了贡献，但是还没进WTF贡献者微信群，请联系我。

2024年WTF Academy在财务上的目标仍是收支平衡。鉴于我们已经入选了optimism的retropgf3和Starknet的EAG资助项目，这意味着在维持收支平衡下，会有更多的资金来资助WTF教程和社区项目。

祝WTF贡献者和成员们2024新年快乐，身体健康，牛市暴富！

**Comment by 0xKaso** [comment]
来自 WTF Academy 新年的温暖。

**Reply by 0xKaso** [reply]
建议：
1.可以定制人均200左右的周边；
2.按照项目曝光/star/重要性等，进行项目资金分发到项目中；

单项目Top3贡献者可申请成为项目管理员，作为项目的唯一对接人，并承担后期项目更新/管理/分配提案等任务。可以单个项目贡献者一个小群，深度研究一个领域，提升质量。

**Reply by AmazingAng** [reply]
好建议

**Reply by keyneszeng** [reply]
这个建议特别好，既能慢慢塑造WTF社区品牌，又能让社区贡献者获得经济收入，并且社区成员后续还能自发发展

**Comment by 0x0077** [comment]
倾向于定制周边

**Comment by 0xLuclee** [comment]
作为Dao资金投资是不是更好一些

**Reply by AmazingAng** [reply]
目前资金量小，投资意义不大

**Comment by web3lm** [comment]
建议弄周边，
这样更有意义

**Comment by XdpCs** [comment]
弄一些衣服的周边 这样穿出去还是活招牌😁
如果有得多，可以发一点点项目资金

**Reply by AmazingAng** [reply]
是的 周边剩下的资金还是会发给贡献者

**Comment by joeyzone** [comment]
我赞同发周边，有意义一些，谈钱伤感情~~

**Comment by leecobaby** [comment]
建议弄个简易的系统，可自己兑换周边，也可自行提现到自己的账户，对于社区长久的发展这是一劳永逸的。

**Reply by AmazingAng** [reply]
有没有类似的开源原型？

**Reply by leecobaby** [reply]
有一些传统的商城积分管理系统开源项目，但我认为并不适合目前的社区，首先要考虑这种系统数据应该如何管理，中性化的还是去中心化的，还需要相应的链上服务合约。

**Reply by AmazingAng** [reply]
感觉早期积分可以先存在链下，然后慢慢上链，先把商城搭起来。

**Comment by chidawen** [comment]
建议弄一些周边，挺好的

**Comment by Asher-Ding** [comment]
周边周边！因为我不是贡献者，但是我想从贡献者里买周边。😄

**Comment by xinparkn** [comment]
周边！目前刚打算加入团伙，还不是贡献者。ｘD
